commons/Jahsohn_Aksot.pl

```diff
@@ -0,0 +1,40 @@
+sub EVENT_SAY {
+	if ($text=~/hail/i) {
+		quest::say("Greetings, $name.");
+	}
+	if ($text=~/rykas gave it to me/i) {
+		quest::say("I have not heard of anyone speaking that name out loud in years. Rykas once shared a tale with me, though I have lived with fear in my mind ever since. He told me afterwards that I did not have the balance I thought I had. Have you come for the Words of Magi'kot?"); 
+	}
+	if ($text=~/words/i) {
+		quest::say("The Words of Magi'kot can only be shared with those elementalists whose minds are of balance. The Words were separated into three pieces by a high council of Magi who feared that the story would bring an end to elementalists throughout Norrath. If you feel you wish to read from the sacred Power of the Orb then an elementalist of such power should easily accomplish this task.");
+	}
+	if ($text=~/what task/i) {
+		quest::say("Bring me the torn pages of Magi'kot. The first section can be found in the depths of a tainted forest, warded by an angry lupine. It is also rumored that one section is found in the haunted estate of a murderer, on a many armed creature. The last may be found in the belly of an amphibian who thirsts for blood.");
+	}
+	if ($text=~/bantil sent me/i) {
+		quest::say("You are working with Bantil? I hope you had some luck finding the books. Let me know if there is anything I can do to help you. Bantil is a friend, despite his gnomish heritage.");
+		quest::setglobal("mage_epic_jah", 1, 5, "F");
```

Comment:
Are we refactoring globals to data buckets while we are at it?

Reply:
Good catch - I'll change to a data bucket.

Reply:
Fixed with aa1d4b5574186f8056ddfeaa953c7fef54efbc79; changed to use data buckets, and also guarded with an expansion check for Omens of War since this dialogue is for the Mage 1.5 epic.